python/py-mayavi/Portfile

```diff
@@ -17,12 +17,12 @@ long_description    3D Scientific Data Visualization and Plotting using VTK as t
 license             BSD
 platforms           darwin
 
-checksums           rmd160  e1228d368978b5abde4ee4946306b82e3431e087 \
-                    sha256  27d8f02cf8f40afce2b6e4dec7de740fbcbcd25d1bb69e26069adb4ab9f4a9a9 \
-    size    9051159
+checksums           rmd160  a78fee221fa1243096101cf088dc26dd14567d5f \
+                    sha256  5dff720e2d3342697d2f6b22621a9a7cf4a34134aceb9bf22446eea2257e4789 \
+                    size    9066423
 
-python.versions     27 35 36 37
-python.default_version      36
+python.versions     27 35 36 37 38
+python.default_version      37
```

Comment:
I was recently instructed, on another PR, not to include `python.default_version` unless there is a good reason. The Python portgroup sets the currently preferred default. So I think the python.default_version should be removed here.

Reply:
Pushed update to branch with suggested change.

Reply:
Pushed update to branch with suggested change.